packages/ui/components/pages/PoolPage/MarketsList/AdditionalInfo/FundButton/WithdrawModal/index.tsx

```diff
@@ -98,7 +98,11 @@ export const WithdrawModal = ({
 
       let resp;
       if (maxWithdrawAmount.eq(amount)) {
-        resp = await currentSdk.withdraw(asset.cToken, constants.MaxUint256);
+        if (asset.borrowBalance.isZero()) {
```

Comment:
the user borrows are very probably in the other markets in the pool... the easiest way to check if the redeem call will pass is to check with `comptroller.redeemAllowed(marketAddress, userAddress, amount)`